specs/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/network/3.1.1/connect.successful.fragmented/client.rpt

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
I think we need to revise the terminology fragmented vs unfragmented for these scenarios.

Lets remove fragmented suffix from the normal case and treat "unfragmented" as the special case.

TLS False Start assumes successful TLS handshake and starts sending encrypted TLS records while confirmation of the successful (or unsuccessful) handshake is still in flight.
https://webopedia.com/definitions/tls-false-start/

Let's reuse that terminology here, giving `connect.subscribe.false.start` as the scenario name, since we send `SUBSCRIBE` assuming a successful `mqtt` handshake while `CONNACK` is still in flight.

Reply:
The normal case (connect.successful) mean that the connect packet came as a whole -> unfragmented.
So when you say: "Lets remove fragmented suffix from the normal case and treat "unfragmented" as the special case." the normal case doesn't have a fragmented suffix. 
For connect.subscribe, I renamed the scripts. Let me know if I understood the first part correctly.